extensions/wikia/EmergencyBroadcastSystem/js/EmergencyBroadcastSystem.js

```diff
@@ -0,0 +1,24 @@
+(function ($) {
+	'use strict';
+
+	function init() {
+		$('.ebs-primary-action').click(onClickYes);
+		$('.ebs-secondary-action').click(onClickNo);
+	}
+
+	function onClickYes() {
```

Comment:
Those two methods: onClickYes and onClickNo are almost identical - I don't thing they are going to change so it might make sense to unify them to one method.